gem/management/commands/remove_empty_nav_tags.py

```diff
@@ -11,9 +12,13 @@ def handle(self, **options):
         for article in ArticlePage.objects.all():
             for nav_tag in article.nav_tags.all():
                 if nav_tag.tag is None:
-                    nav_tag.delete()
-
-            if article.live:
-                article.save_revision().publish()
-            else:
-                article.save_revision()
+                    try:
+                        nav_tag.delete()
```

Comment:
I don't think this should be in the try block. Only the things that you expect to throw the error should be covered by it